wercker.yml

```diff
@@ -89,7 +89,7 @@ build:
         code: go/build.sh
     - script:
         name: test go koding config
-        code: scripts/wercker/run-command ./run exec go test -v --race koding/kites/config
+        code: scripts/wercker/run-command ./run exec go test -v --race -coverprofile=profile.out -covermode=atomic koding/kites/config
```

Comment:
you could use `scripts/wercker/run-command /scripts/gotests.sh koding/kites/config`